### PR TITLE
enable travis

### PR DESCRIPTION
**Pull request by patriknw**
just `test:compile` for now

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: scala
+sudo: false
+scala:
+  - "2.11.7"
```
Comment (ktoso) [inline .travis.yml]: could bump to 2.11.8

Comment (patriknw) [inline .travis.yml]: ok, updated

**Comment by patriknw** [comment]
enabled tests on travis, with get and run of kafka/zookeeper

**Comment by 13h3r** [comment]
Should we also add some doc how to use travis build locally?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+scala:
+  - "2.11.8"
+jdk:
+  - oraclejdk8
+before_install:
+- wget http://apache.mirrors.spacedump.net/kafka/0.9.0.1/kafka_2.11-0.9.0.1.tgz -O kafka.tgz
+- mkdir -p kafka && tar xzf kafka.tgz -C kafka --strip-components 1
+- nohup bash -c "cd kafka && bin/zookeeper-server-start.sh config/zookeeper.properties &"
+- nohup bash -c "cd kafka && bin/kafka-server-start.sh config/server.properties &"
+- sleep 5
```
Comment (ktoso) [inline .travis.yml]: we could put that in a script, call the script from here – then people could use it locally too.

**Comment by ktoso** [comment]
Good point @13h3r though let's go the direction of https://github.com/akka/reactive-kafka/pull/105/files#r55833791 maybe?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+language: scala
+sudo: false
+scala:
+  - "2.11.8"
+jdk:
+  - oraclejdk8
+before_install:
+- wget http://apache.mirrors.spacedump.net/kafka/0.9.0.1/kafka_2.11-0.9.0.1.tgz -O kafka.tgz
+- mkdir -p kafka && tar xzf kafka.tgz -C kafka --strip-components 1
+- nohup bash -c "cd kafka && bin/zookeeper-server-start.sh config/zookeeper.properties &"
```
Comment (13h3r) [inline .travis.yml]: I am not familiar with travis, but if we started it here then where we plan stop it?

Comment (ktoso) [inline .travis.yml]: you don't. the container gets killed.

Comment (13h3r) [inline .travis.yml]: Got it. Thanks

File: .travis.yml (hunk 2 above)
Comment (13h3r) [inline .travis.yml]: then we also provide stop script too

Comment (ktoso) [inline .travis.yml]: true

**Pull request merged**